.gitlab/internal_kubernetes_deploy/rc_kubernetes_deploy.yml

```diff
@@ -11,9 +11,8 @@ rc_kubernetes_deploy:
       when: manual
   needs:
     - job: docker_trigger_internal
-      artifacts: false
```

Comment:
Is there an artifact generated by this job? I thought it only pushed images

Reply:
There are none, that's why I removed this line. It was there from the past, copy-paste probably.

Reply:
But then it will now attempt to download artifacts, even though there are none.
It doesn't change anything functionally, but I find the intent to be clearer with the explicit `artifacts: false`

Reply:
Oh, I actually could have made a mistake here. I though that skipping that field in the needs section is like setting it to false. But I reread the documentation and it's actually opposite. I will change it back. Sorry for the confusion.